__tests__/app.test.js

```diff
@@ -173,3 +172,34 @@ describe("PATCH /api/articles/:article_id", () => {
       })
   });
 })
+
+describe("GET /api/users", () => {
+  test("Status 200", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+  });
+  test("returns an array of objects, each with the property 'username'", () => {
+      return request(app)
+          .get("/api/users")
+          .expect(200)
+      .then((response) => {
+          expect(Array.isArray(response.body)).toBe(true)
+          response.body.forEach((user) => {
+            expect(user).toEqual(
+              expect.objectContaining({
+                username: expect.any(String)
+              })
+            )
+          })
+      })
+  })
+  test("Status 404 when path entered incorrectly'", () => {
```

Comment:
You shouldn't need to do this test for each endpoint as it's specifically for other endpoints that you do not accept. If you have one test of this nature in your test suite that will be sufficient
Also, I would assert the message you send on the _**response body**_ contains the text you're after rather than accessing `response.error.text`